Harvest.Web/ClientApp/src/Shared/ProjectHeader.tsx

```diff
@@ -7,6 +9,25 @@ interface Props {
 
 export const ProjectHeader = (props: Props) => {
   const { project, title } = props;
+  const [modal, setModal] = useState<boolean>(false);
+  const toggleModal = () => setModal(!modal);
+
+  // Function to show a truncated version of project.requirements if the
+  // character count is above 256
+  const projectReqs = (requirements: string) => {
```

Comment:
Also here, this is pretty minor but you are recalculating this every time the component runs although the requirements prop isn't going to change.  It'd be a good place to `useMemo`